src/docs/Foundations/Typography/BodyStyles/index.mdx

```diff
@@ -6,22 +6,25 @@ status:
   scss: 'wip'
 ---
 
-> The body patterns provide basic typographic styles for readable multilines text or paragraphs like a product description or an article.
+> A body style in typography is the main text part of a pattern or a page. It must be identity and provide basic typographic styles for readable multilines texts or paragraphs.
 
-Body styles are patterns meant to help you use consistent typographic hierarchy with the context and across responsive viewports. It's mainly reserved for users wishing to focus on tasks with added value and user experience. Text classes are families of several font-sizes and help create a hierarchy of contents.
+## Our recommendation
```

Comment:
Sorry but can we rephrase by: 
## Recommendation

Reply:
@nherwegh fixed